### DISCUSSION TITLE
建议支持显示国旗emoji

### DISCUSSION THREAD

**Discussion by st-youth** [zzzgydi/clash-verge#298] [Ideas]
鉴于Verge没有检测代理服务器IP地理位置的功能，可否支持一下国旗emoji，这样可以自己在订阅里加入emoji方便区别。谢谢。

**Comment by zzzgydi** [comment]
windows的不支持显示emoji，是windows字体问题，你换一个支持emoji的字体就好

参考 https://github.com/zzzgydi/clash-verge/issues/50

**Reply by st-youth** [reply]
感谢！
